docs/tm/udq_actionx/pyaction.tex

```diff
@@ -96,7 +96,9 @@ \section{The \pyaction{} keyword}
  
 \subsection{The different arguments}
 The \inlinecode{run()} function will be called with exactly five arguments which
-your implementation can use. The five arguments are:
+your implementation can use. These arguments point to datastructures in the
+simulator, and is the way to interact with the state of the simulation. The five
```

Comment:
is ->are?

Reply:
I have a weak preference for is - I think it points to "the way" - and hence singular?

Reply:
this is the way...?